html/semantics/forms/the-button-element/button-setcustomvalidity.html

```diff
@@ -1,43 +1,17 @@
 <!DOCTYPE HTML>
-<html>
- <head>
-  <title>Forms</title>
-  <script src="/resources/testharness.js"></script>
-  <script src="/resources/testharnessreport.js"></script>
- </head>
- <body>
-  <p>
-    <h3>button_setCustomValidity</h3>
-  </p>
+<title>button setCustomValidity</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
 
-  <hr>
+<button id='test'></button>
```

Comment:
nit: Please avoid the ID 'test'.  Both of this element and testharness's test() can be 'test' property of 'window'.  It's confusing.

Reply:
@tkent-google, many thanks for your comments, I've addressed it, PTAL.